### ISSUE TITLE
Redundant error messages

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#258]
Hi Son,

at a lot of positions you apply the same error message to both `logger.error()` and `BuiltIn().unknown`:

```
logger.error(self.sLoadedCfgError)
BuiltIn().unknown(self.sLoadedCfgError)
```

This causes unnecessary redundancy in console and in log file:

```
ERROR - Verification against JSON schema failed: 'Additional properties are not allowed ('IAMINVALID' was unexpected)'
          Please put the additional params into 'params': { 'global': {...}
UNKNOWN - Verification against JSON schema failed: 'Additional properties are not allowed ('IAMINVALID' was unexpected)'
          Please put the additional params into 'params': { 'global': {...}
```

This is confusing and makes no sense. It is possible to split available information in a more meaningful way. And a generic function should be used for logging UNKNOWN tests (and the same for failed tests).

Something like:

```
def log_unknown(error=None, impact=None):
    default_error  = "An unknown error happened."
    default_impact = "The test execution will be aborted now."
    if error is None:
        error = default_error
    else:
        error = f"{error}"
    if impact is None:
        impact = default_impact
    else:
        impact = f"{impact}"
    logger.error(error)
    BuiltIn().unknown(impact)
```

Now you can do this:

```
impact = "Unable to load the test configuration. The test execution will be aborted now."
log_unknown(self.sLoadedCfgError, impact)
```

In log file this looks much more plausible now:

```
ERROR - Verification against JSON schema failed: 'Additional properties are not allowed ('IAMINVALID' was unexpected)'
          Please put the additional params into 'params': { 'global': {...}
UNKNOWN - Unable to load the test configuration. The test execution will be aborted now.
```

**Comment by namsonx** [comment]
Hello Holger, 

I pushed [commit b3a65f87c05](https://github.com/test-fullautomation/robotframework-testsuitesmanagement/commit/b3a65f87c05ec50794ca4d04a322ebba73b09e6c) to improve the method of logging error messages when loading the configuration failed.
I also pushed [commit bed8201f9](https://github.com/test-fullautomation/robotframework-testsuitesmanagement/commit/bed8201f9cab57a4af547834a99fe2e0a5ddeb87) to update reference log for the Selftest.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
Released with RobotFramework AIO 0.12.0
